Mf2/Parser.php

```diff
@@ -1141,6 +1141,10 @@ public function parseH(\DOMElement $e, $is_backcompat = false, $has_nested_mf =
 			'type' => $mfTypes,
 			'properties' => $return
 		);
+		
+		if($e->getAttribute("id")) {
```

Comment:
The `id` could be just the string `0`. E.g. on a page that assign counters to something. This would fail currently as that string is considered falsy by PHP. Instead it should probably explicitly check against empty strings:

```suggestion
		if($e->getAttribute('id') !== '') {
```